framework/web/AssetManager.php

```diff
@@ -316,14 +316,20 @@ protected function loadDummyBundle($name)
      * The actual URL is obtained by prepending either [[AssetBundle::$baseUrl]] or [[AssetManager::$baseUrl]] to the given asset path.
      * @param AssetBundle $bundle the asset bundle which the asset file belongs to
      * @param string $asset the asset path. This should be one of the assets listed in [[AssetBundle::$js]] or [[AssetBundle::$css]].
+     * @param bool|null $appendTimestamp Whether to override manager's appendTimestamp property
```

Comment:
```suggestion
     * @param bool|null $appendTimestamp Whether to append timestamp to the URL.
```

Reply:
Well, if it's null, the manager's value takes over so that change might be confusing

Reply:
hmm, ok then :P